src/sbt-test/sbt-scalafix/cross-build/build.sbt

```diff
@@ -22,7 +22,7 @@ lazy val root = project
   )
 
 lazy val scala210 = project.settings(
-  scalaVersion := "2.10.5"
+  scalaVersion := "2.10.7"
```

Comment:
prevents `error while loading package, Missing dependency 'object java.lang.Object in compiler mirror', required by /home/runner/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.10.5.jar(scala/package.class)` on CI